site/blog/2023/v2.11.md

```diff
@@ -0,0 +1,23 @@
+---
+title: v2.11
+date: 2023-01-13
+tags:
+    - releases
+---
+
+## [v2.11](https://github.com/saulpw/visidata/releases/tag/v2.11) (2023-01-13)
+
+Hi, friends of VisiData!
+
+I’m happy to announce a new maintenance release 2.11. This will be the last release before some big, exciting changes introduced in 3.0.
+
+The most significant change is that 2.11 *drops support for Python 3.6* and *adds support for Python 3.11*.
+
+2.11 also includes several other small improvements and bug fixes, of which I’ll highlight two:
```

Comment:
This release is a bugfixing machine by Saul! I wish we had the count for how many issues we closed in the past 2-3 weeks. I think I want to emphasise that.

Reply:
Are there specific bugs in the changelog that would be good to highlight? (Anything that's been bothering some users?)